kubeflow/core/prototypes/ambassador.jsonnet

```diff
@@ -3,7 +3,7 @@
 // @description Ambassador Component
 // @shortDescription Ambassador
 // @param name string Name
-// @optionalParam cloud string null Cloud
+// @optionalParam platform string none supported platforms {none|gke|minikube}
```

Comment:
Was @ashahba 's PR already submitted? Why change "null" to "none"? Is that to match what kfctl does?

Reply:
@ashahba's PR missed optionalParam. Yes - changed to 'none' for kfctl.